chain/chain/src/test_utils.rs

```diff
@@ -1026,6 +1026,7 @@ impl RuntimeAdapter for KeyValueRuntime {
             current_proposals: vec![],
             prev_epoch_kickout: vec![],
             epoch_start_height: 0,
+            epoch_height: 1,
```

Comment:
It is an unfortunate naming collision here (`epoch_start_height` is a block height)